solc/CommandLineInterface.cpp

```diff
@@ -1037,7 +1037,7 @@ General Information)").c_str(),
 		)
 		(
 			g_argOptimizeRuns.c_str(),
-			po::value<unsigned>()->value_name("n")->default_value(200),
+			po::value<size_t>()->value_name("n")->default_value(OptimiserSettings{}.expectedExecutionsPerDeployment),
```

Comment:
I just found https://github.com/nomiclabs/hardhat/issues/1589, which made me realize that this changes the maximum accepted value from `2**32-1` to `2**64-1` on 64-bit platforms. I need to adjust the Standard JSON part to match. Currently it uses `isUint()` which still enforces the `2**32-1 limit`:

https://github.com/ethereum/solidity/blob/cbf1c3ae6969781e5ce050adb82129b4d9379fa8/libsolidity/interface/StandardCompiler.cpp#L572-L577

Reply:
I restored `unsigned` in this one place to keep the validation consistent between CLI and Standard JSON and not disrupt this PR too much. As is it now at least keeps the status quo of `size_t` internally and 2**32-1 in validations. We can change the type in a separate PR.